README.md

```diff
@@ -53,6 +53,38 @@ You can first localize the robot by initializing the pose and then setting a nav
 
 [![](media/localize.png)](media/localize.mp4)
 
+## Running the reconnaissance demos:
+
+### Host/container insider attacker
+``` bash
+rocker rosswg/turtlebot3_demo "byobu -f configs/unsecure.conf attach"
```

Comment:
```suggestion
rocker rosswg/turtlebot3_demo:roscon19 "byobu -f configs/unsecure.conf attach"
```

Reply:
Let's stay consistent and point the users to use the reserved `roscon19` tag from the docker repo to ensure they stick with the curated image for the tutorial.